### ISSUE TITLE
C++ port/fork

### ISSUE THREAD

**Issue by MrSapps** [analgesicproductions/Anodyne-1-Repo#1]
Seems a few people including myself would be interested in this. Figured someone should create an issue to avoid any duplicated work and open up discussion?

My idea was to port to PS1 which is what a C++ port would open up. However it would also require a mass optimisation of the assets. Which would also benefit other platforms in terms of less ram/vram usage.

**Comment by lifning** [comment]
agreed - i personally think this game belongs on the GBA (having itself been inspired by a GB Zelda game), which would possibly require an altogether different type of asset optimization than the PSX would, GPU-wise - though if memory serves, the usual pattern for devkitARM/libgba projects is to process assets at build time (e.g. png into tile/palette data)

**Comment by Lenkense** [comment]
Sorry to bother, has anyone started coding this idea for a project?

**Comment by analgesicproductions** [comment]
I don't believe so!

**Comment by MrSapps** [comment]
https://github.com/MrSapps/AnodynePSX started prototyping here
